mpi-proxy-split/util/mpi-logger/README

```diff
@@ -0,0 +1,17 @@
+## MPI-LOGGER
+The purpose of mpi-logger is to intercept MPI calls from native program without involving running MANA.
+Using this tool people can compare MPI arguments between MANA and native run, in order to debug MANA.
```

Comment:
`can compare MPT arguments` -> `can compare arguments to MPI functions`
`and native run` -> `and a native run`

I suggest adding the following sentence:
> In situations where the MPI application is close to deterministic (e.g., one rank, or multiple ranks communicating in a deterministic pattern), this tool helps find where the calls to MPI functions in MANA don't reflect the correct semantics.